lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -340,7 +340,7 @@ def _on_scrape_targets_changed(self, event):
 DEFAULT_RELATION_NAME = "metrics-endpoint"
 RELATION_INTERFACE_NAME = "prometheus_scrape"
 
-DEFAULT_ALERT_RULES_RELATIVE_PATH = "./prometheus_alert_rules"
+DEFAULT_ALERT_RULES_RELATIVE_PATH = "./src/prometheus_alert_rules"
```

Comment:
Originally it was `src` but was changed by a recent commit from @mmanciop . I am approving PR but double check with him.